### PR TITLE
add processing for dcatus rights; add user constraint; format and upd…

### PR DESCRIPTION
**Pull request by rshewitt**
related to [#4898](https://github.com/GSA/data.gov/issues/4898)
- updated how the dcatus `rights` writer processes data. the reason for this is to be compatible with all of the metadata types we currently harvest ( i.e. fgdc, iso19115-2, and iso19139 ). without the change only iso19115-3 can be processed. 
- add user constraint; format and update fixtures and some assertions;

### CODE REVIEW

File: test/translator/testData/iso19115-3-to-dcatus.json
Comment (btylerburton) [inline test/translator/testData/iso19115-3-to-dcatus.json:1]: Not sure what's changed here. Formatting?

**Review by btylerburton — APPROVED** [review]
Nice that I'm starting to see how this stuff fits together. I could that as a win!

File: test/translator/testData/iso19115-3-to-dcatus.json
Comment (rshewitt) [reply test/translator/testData/iso19115-3-to-dcatus.json:1]: `rights` changed. the file is 1 line so it's confusing. let me change this.

Comment (rshewitt) [reply test/translator/testData/iso19115-3-to-dcatus.json:1]: ahhh lemme do it in my next pr

**Pull request merged**